manim/mobject/svg/tex_mobject.py

```diff
@@ -48,13 +48,20 @@ class SingleStringMathTex(SVGMobject):
         "organize_left_to_right": False,
         "alignment": "",
         "type": "tex",
+        "template": None,
     }
 
     def __init__(self, tex_string, **kwargs):
         digest_config(self, kwargs)
+        if self.template is None:
```

Comment:
I suggest we should avoid using `CONFIG` simple change would be to add it as a parameter to `__init__`.
```suggestion
    }

    def __init__(self, tex_string, template = None, **kwargs):
        digest_config(self, kwargs)
        if self.template is None:
```

Also, why aren't we statically typing it?

Reply:
Where is the documentation for the `template` parameter, should it be a string?

Reply:
the template parameter should be a `TexTemplate()`.
Currently every `SingleStringTexMobject` uses the `global config["tex_template"]`
All this does is allow you to set it on a per mobject basis so you can write your own derived classes
`MyTexMobject` that uses its own template.

Since in general we want to get away from CONFIG, perhaps I can take that up again in  #483

Reply:
Ah. I see your suggestion.

To be honest, I had simply put it in CONFIG because I had been studying 3b1b code and just took over the practice from there. 
I'll accept your change.